Tests/Marketplace/upload_packs.py

```diff
@@ -199,6 +201,47 @@ def update_index_folder(index_folder_path, pack_name, pack_path, pack_version=''
         return task_status
 
 
+def clean_non_existing_packs(index_folder_path, private_packs, storage_bucket):
```

Comment:
can you create some unit tests for this?

Reply:
Yes

Reply:
@yuvalbenshalom  Added unit tests. Note that now`clean_non_existing_packs` function has detection that checks environment variables (circle CI env variables). And in order to mock it, I've used ` @patch.dict('os.environ',..` https://docs.python.org/3.4/library/unittest.mock.html#unittest.mock.patch.dict
Please take a look and let me know if it is o.k to use it.